pyproject.toml

```diff
@@ -44,3 +44,5 @@ dev = [
     "ipdb>=0.13.13",
     "pre-commit>=3.7.1",
 ]
+[project.scripts]
+mllam_data_prep = "mllam_data_prep:cli.call"
```

Comment:
Ah! I hadn't spotted this before. Ok, so there's actually three changes here, this is a really cool addition. Should we maybe mention this in the README too?

Reply:
I updated the README